backend/blueprint/api/controllers/loginUser.py

```diff
@@ -45,3 +47,36 @@ def login():
     return jsonify(
         Response(403).generateMessage(
             'No name key received'))
+
+
+@router.route('/get_user', methods=['GET'])
+def index():
+    data = request.args
+    user = User.getUser(username=data.get('username'))
+    schema = UserSchema()
+    return jsonify(schema.dump(user).data)
+
+
+@router.route('/register', methods=['POST'])
+def po():
```

Comment:
what is `po`? can you please name functions something more relevant and human readable